src/App.re

```diff
@@ -25,9 +25,12 @@ let make = _children => {
       (
         switch (self.state.route) {
         | Login => <Login submitHandler=(_event => self.send(Login)) />
-        | JobApp => <JobApp submitHandler=(_event => self.send(Logout))
-                            signOutHandler=(_event => self.send(Logout)) />
+        | JobApp =>
+          <JobApp
+            submitHandler=(_event => self.send(Logout))
+            signOutHandler=(_event => self.send(Logout))
+          />
         }
       )
     </div>,
-};
+};
```

Comment:
minor comment:
I think you haven't wired up VSCode to apply the `.editorconfig` because it should add a newline at the end of each file: `insert_final_newline = true`

Reply:
not indispensable, it's just standard linting. As I said, not an issue per see.
If you want to set it up you just gotta download [this plugin](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig)